main.yml

```
@@ -41,12 +41,26 @@ ansible:
   title: Ansible Automation Platform
   frontend:
     sub_apps:
+      - id: ansible-dashboard
       - id: automation-hub
         default: true
       - id: catalog
       - id: automation-analytics
   top_level: true
 
+ansible-dashboard:
+  title: Overview
+  deployment_repo: https://github.com/RedHatInsights/ansible-platform-dashboard-build
+  frontend:
+    module:
+      appName: ansible-dashboard
+      scope: ansibleDashboard
+      module: ./RootApp
+    paths:
+      - /ansible
+      - /ansible/ansible-dashboard
```

If this app is not the default, this needs to be updated
```suggestion
    paths:
      - /ansible/ansible-dashboard
```